conans/client/build/cmake_flags.py

```diff
@@ -172,7 +172,7 @@ def _get_cpp_standard_vars(self):
         definitions["CONAN_STD_CXX_FLAG"] = cppstd_flag(self._conanfile.settings)
         return definitions
 
-    def _cmake_cross_build_defines(self):
+    def _cmake_cross_build_defines(self, cmake_version=None):
```

Comment:
```suggestion
    def _cmake_cross_build_defines(self, cmake_version):
```

No need to add a default value. The call to `CMake::get_version()` returns the value or raises. It also simplifies the implementation.